memberlist.go

```diff
@@ -233,6 +233,7 @@ func newMemberlist(conf *Config) (*Memberlist, error) {
 	go m.streamListen()
 	go m.packetListen()
 	go m.packetHandler()
+	go m.checkQueueDepth("Intent", m.broadcasts)
```

Comment:
- Shall we use all lower-case for the name like some other places?

https://github.com/hashicorp/memberlist/blob/e6ff9b2d87a3f0f3f04abb5672ada3ac2a640223/state.go#L295-L297

- Also I'm not sure if `Intent` is a good name here. "Intent" is consistent with the name in serf, but make it difficult to infer in the memberlist library. If we choose a different name like "broadcast" or "transmit", then we may need to update the consul telemetry page about the correlation between memberlist.broadcase and serf.Intent.

Reply:
Great callout.  I've changed the name and signature to `checkBroadcastQueueDepth()` and within the function it passes `broadcast` as the metric name.  Thank you for calling this out.